.github/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 8. Enhancement - Re-structured 'Misc' folder - thanks to @PLM1995 (Peter Mooney)
 9. AIRAC (2405) - Updated Fairford (EGVA) Runway coordinates - thanks to @danielbutton (Daniel Button)
 10. Bug - Fixed Fairford (EGVA) ATZ coordinates - thanks to @danielbutton (Daniel Button)
+11. Procedure Change - Updated agreed level (EHAM departure via LONAM) - thanks to @khardern
```

Comment:
```suggestion
11. Procedure Change (2405) - Updated agreed level (EHAM departure via LONAM) - thanks to @khardern
```
Per [style guide](https://github.com/VATSIM-UK/UK-Sector-File/blob/main/.github/Style%20Guide.md#entry-type-), AIRAC and Procedure Changes should include the AIRAC which they apply to/from.